sae.c

```diff
@@ -2253,7 +2287,11 @@ JC: Commented out until we decide whether this is needed (in which case we must
 #endif
     TAILQ_INIT(&peers);
     TAILQ_INIT(&blacklist);
-    RAND_pseudo_bytes((unsigned char *)&token_generator, sizeof(unsigned long));
+    i = RAND_bytes((unsigned char *)&token_generator, sizeof(unsigned long));
+    if (i == 0) {
```

Comment:
what is this return value, number of bytes generated?  should we check that i == sizeof(unsigned long) instead?

Reply:
It's 1 on success, 0 on failure.  https://www.openssl.org/docs/man1.1.0/crypto/RAND_bytes.html

Reply:
ah right, openssl has terrible ideas about return values... https://www.openssl.org/news/secadv/20090107.txt and I seem to remember others.